src/common/lib/client/connection.ts

```diff
@@ -59,7 +59,9 @@ class Connection extends EventEmitter {
   }
 
   get recoveryKey(): string | null {
-    Logger.deprecated('Connection.recoveryKey attribute', 'Connection.createRecoveryKey() method');
+    Logger.deprecationWarning(
```

Comment:
Isn't it better to use `Logger.deprecated` for deprecation messages? It adds `is deprecated and will be removed in a future version` automatically so there is no difference in wording between deprecation messages. Based on the recent commits it seemed to me like that was the intention.
So it would be
```javascript
Logger.deprecated(
  'The `Connection.recoveryKey` attribute',
  'It has been replaced by the `Connection.createRecoveryKey()` method. Replace your usage of `recoveryKey` with the return value of `createRecoveryKey()`.'',
)
```

Reply:
I see, so basically you choose .deprecated / .deprecationWarning / .renamedClientOption / .renamedMethod whichever sound best in the current context.

SGTM

Reply:
Yeah, exactly! It's all pretty subjective, and depending on the time of day and phase of the moon I'd probably be easily convinced that what you suggested actually sounds better 🤷